apps/mobile/src/app/(home)/mpc-wallets.tsx

```diff
@@ -0,0 +1,110 @@
+import { Trans, t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import {
+  Box,
+  Cell,
+  LogoMPCBitgo,
+  LogoMPCCapsule,
+  LogoMPCCopper,
+  LogoMPCFireblocks,
+  LogoMPCFordefi,
+  LogoMPCPortal,
+  LogoMPCPrivy,
+  LogoMPCQredo,
+  QuestionCircleIcon,
+  SheetRef,
+  Text,
+  Theme,
+  TouchableOpacity,
+} from '@leather.io/ui/native';
+
+function getUnavailableFeatures() {
+  return {
+    bitgo: { title: t`BitGo`, Icon: LogoMPCBitgo },
+    capsule: { title: t`Capsule`, Icon: LogoMPCCapsule },
+    copper: { title: t`Copper`, Icon: LogoMPCCopper },
+    fireblocks: { title: t`Fireblocks`, Icon: LogoMPCFireblocks },
+    foredefi: { title: t`Foredefi`, Icon: LogoMPCFordefi },
+    portal: { title: t`Portal`, Icon: LogoMPCPortal },
+    privy: { title: t`Privy`, Icon: LogoMPCPrivy },
+    qredo: { title: t`Qredo`, Icon: LogoMPCQredo },
+  };
+}
+
+export default function MPCWalletsScreen() {
+  const { bottom } = useSafeAreaInsets();
+  const theme = useTheme<Theme>();
+  const notifyUserSheetRef = useRef<SheetRef>(null);
+  const [optionData, setOptionData] = useState<OptionData | null>(null);
+  function onOpenNotificationsSheet(option: OptionData) {
+    setOptionData(option);
+    notifyUserSheetRef.current?.present();
+  }
+  function onCloseNotificationsSheet() {
+    setOptionData(null);
+  }
+
+  return (
+    <>
+      <Box
+        flex={1}
+        backgroundColor="ink.background-primary"
+        style={{ paddingBottom: bottom + theme.spacing['5'] }}
+      >
+        <ScrollView contentContainerStyle={{ paddingHorizontal: theme.spacing['5'] }}>
+          <Box gap="3" pt="5">
+            <TouchableOpacity
+              onPress={() => {
+                // TODO: show some kind of a helper here
+              }}
+              p="5"
+              position="absolute"
+              right={-theme.spacing['5']}
+              zIndex={10}
+              top={theme.spacing['1']}
+            >
+              <QuestionCircleIcon color={theme.colors['ink.text-primary']} variant="small" />
+            </TouchableOpacity>
+            <Box>
+              <Trans>
+                <Text variant="heading03">CONNECT</Text>
+                <Text variant="heading03">MPC WALLET</Text>
+              </Trans>
+            </Box>
+            <Box gap="1" pt="5">
+              {Object.entries(getUnavailableFeatures()).map(featureEntry => {
+                const [featureKey, feature] = featureEntry;
+                const mpcWalletName = feature.title;
+                function onPress() {
+                  onOpenNotificationsSheet({
+                    title: t`Connect MPC wallet: ${mpcWalletName}`,
+                  });
+                }
+                return (
+                  <Cell
+                    py="4"
+                    key={featureKey}
+                    onPress={onPress}
+                    title={feature.title}
+                    Icon={feature.Icon}
```

Comment:
To follow our code conventions, we should start refactoring capitalized props in mobile? They should all be lowercase.

Reply:
I think that for `Icon`, the way things are setup, it needs to be passed as upper case as it's a component and sometimes we need to pass more props to it. 

We could call it `icon` and load it with `{icon}` but then if we need to set the color it needs to be capital to give it props `<Icon color={...`. Maybe there is a way to do this while keeping it lower case.

Reply:
If we are refactoring away `Cell` maybe we don't need to do this anymore? 

In there we are using it: 
```
   {Icon && (
          <Box flexDirection="row" p="2" bg={getIconBackgroundColor(_variant)} borderRadius="round">
            <Icon color={theme.colors[getIconColor(_variant)]} />
          </Box>
        )}
```

Maybe we can instead just pass the icon with the colour needed?

Reply:
Yeah, I refactored in some places so we can pass as lowercase. We need to do it in all places when encountered. The icons can just be type `ReactNode` now.